tel2puml/tel2puml_types.py

```diff
@@ -126,9 +128,80 @@ class OtelPVOptions(TypedDict):
     ingest_data: bool
 
 
-class PVPumlOptions(TypedDict, total=False):
+class PVPumlOptions(TypedDict):
     """Typed dict for options for pv_to_puml"""
 
     file_list: list[str]
     job_name: str
     group_by_job_id: bool
+
+
+class OtelArgs(BaseModel):
```

Comment:
I think just one class called OtelToPVArgs.

Can we have an extra field called `command` and then model validation to make sure that if command is "otel2puml" save-events must be False

Reply:
Since --save-events is only added to otel_to_pv_parser, if we call otel2puml with -se flag, it will error out saying -se not a valid argument. So by having save_events default to False we shouldn't require the command and validation check

Reply:
That's correct but without that flowing into it and in isolation that wouldn't be the case so from a test perspective I just wanted to have exactly only the possibilities that we should have so we can be sure by testing those functions + models that our output is going to be what we expect